### DISCUSSION TITLE
My website looks different on windows 10

### DISCUSSION THREAD

**Discussion by vineetver** [mmistakes/minimal-mistakes#2654] [Q&A]
This is what my theme looks like during development on my mac and linux machine. 

And this is what my theme looks like on windows 10. My sidebar have no links, share doesn't work and my nav links are somehow italic.

Link to my code:
[github](https://github.com/vineetver/minimal-mistakes-clone)

**Comment by mmistakes** [comment]
Looks like Internet Explorer might be running in compatibility mode. A dead give away is usually the italics type. This happens because it’s knocking IE down to version 8 which doesn’t support web fonts and other CSS3 properties.

**Comment by mmistakes** [comment]
What browser are you using by the way? You don’t have any sort of content/ad blocker plugins enabled do you? Might be blocking third party assets like Google fonts and or Font Awesome which does all the author sidebar social links.

**Comment by vineetver** [comment]
I'm using Chrome on all my machines, oh crap it might be ad guard blocking the content. I'll try disabling it, i have a really aggressive ad block set up on my windows machine :(

**Comment by vineetver** [comment]
The problem has been resolved, thank you so much. Adguard and Kaspersky were causing the problem

**Comment by vineetver** [comment]
I have another question, I can't seem to override this transition of nav links.

<img width="162" alt="Screenshot 2020-08-14 at 2 00 40 PM" src="https://user-images.githubusercontent.com/66165922/90230048-948b9f80-de36-11ea-960a-e2b4260892e7.png">

**Comment by mmistakes** [comment]
The nav link underline transition is applied to a `:before` pseudo class. Relevant CSS is here: https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_navigation.scss#L233-L252
